examples/mysql/docker/docker-compose.yaml

```diff
@@ -12,26 +12,18 @@ services:
     ports:
       - "3306:3306"
 
-# Jaeger
-
-  jaeger-all-in-one:
-    image: jaegertracing/all-in-one:latest
-    ports:
-      - "16686:16686"
-      - "14268"
-      - "14250"
-
 # Zipkin
 
   zipkin-all-in-one:
     image: openzipkin/zipkin:latest
+    platform: linux/amd64
```

Comment:
Why add this platform tag ? Unlike mysql there are arm64 images (and they work fine on my M1 laptop)

Reply:
@raphael-theriault-swi   Right you are. Thanks.

```
% docker buildx imagetools inspect openzipkin/zipkin:latest
Name:      docker.io/openzipkin/zipkin:latest
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:32599e9a997272a15a57f1775709ad86f973de98b407b9a1cbcfcb0d9b60d23b

Manifests:
  Name:      docker.io/openzipkin/zipkin:latest@sha256:f4875f535382664b33c76cc2d31e4d5c240571c811e91a08962a63d845ed30f1
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64

  Name:      docker.io/openzipkin/zipkin:latest@sha256:29a4743a1fe7c7edb8d152f35f113cfffc437024b21decf2e2ba5d4050da9902
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64

  Name:      docker.io/openzipkin/zipkin:latest@sha256:44f7e529f12b75cf1a76955eda688dcf06ffef48f574aea07dcb3174a366a327
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/ppc64le

  Name:      docker.io/openzipkin/zipkin:latest@sha256:2855f9743a829f0621280d8e035342994d72f15e8ef740ec2b3d17aeaa5df183
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/s390x
```

I've dropped that in commit 95bf3f0c
I added that too hastily when adding it for mysql.